l10n_co_res_partner/models/l10n_co_res_partner.py

```diff
@@ -0,0 +1,459 @@
+        ('ident_unique',
+         'UNIQUE(doctype,xidentification)',
+         "Identification number must be unique!"),
+    ]
+
+    # Check to handle change of Country, City and Municipality
+    change_country = fields.Boolean(string="Change Country / Department?",
+                                    default=True, store=False)
+
+    # Name of point of sales / delivery contact
+    pos_name = fields.Char("Point of Sales Name")
+
+    # Birthday of the contact (only useful for non-company contacts)
+    xbirthday = fields.Date("Birthday")
+
+    @api.depends('xidentification')
+    def _compute_concat_nit(self):
+        """
+        Concatenating and formatting the NIT number in order to have it
+        consistent everywhere where it is needed
+        @return: void
+        """
+        # Executing only for Document Type 31 (NIT)
+        for partner in self:
+            if partner.doctype is 31:
+                # First check if entered value is valid
+                self._check_ident()
+                self._check_ident_num()
+
+                # Instead of showing "False" we put en empty string
+                if partner.xidentification is False:
+                    partner.xidentification = ''
+                else:
+                    partner.formatedNit = ''
+
+                    # Formatting the NIT: xx.xxx.xxx-x
+                    s = str(partner.xidentification)[::-1]
+                    newnit = '.'.join(s[i:i+3] for i in range(0, len(s), 3))
+                    newnit = newnit[::-1]
+
+                    nitList = [
+                        newnit,
+                        # Calling the NIT Function
+                        # which creates the Verification Code:
+                        self._check_dv(str(partner.xidentification))
+                    ]
+
+                    formatedNitList = []
+
+                    for item in nitList:
+                        if item is not '':
+                            formatedNitList.append(item)
+                            partner.formatedNit = '-' .join(formatedNitList)
+
+                    # Saving Verification digit in a proper field
+                    self.dv = nitList[1]
+
+    @api.onchange('name1', 'name2', 'lastname1', 'lastname2', 'companyName',
+                  'pos_name')
+    def _concat_name(self):
+        """
+        This function concatenates the four name fields in order to be able to
+        search for the entire name. On the other hand the original name field
+        should not be editable anymore as the new name fields should fill it up
+        automatically.
+        @return: void
+        """
+        # Avoiding that "False" will be written into the name field
+        if self.name1 is False:
+            self.name1 = ''
+
+        if self.name2 is False:
+            self.name2 = ''
+
+        if self.lastname1 is False:
+            self.lastname1 = ''
+
+        if self.lastname2 is False:
+            self.lastname2 = ''
```

Comment:
Why this?

It is a bad practice, the proper way is have the False values, can you extend why this is needed for you?